src/Uno.UI/UI/Xaml/VisualStateGroup.cs

```diff
@@ -141,6 +141,28 @@ private void VisualStateChanged(object sender, IVectorChangedEventArgs e)
 			RefreshStateTriggers();
 		}
 
+		internal void ReevaluateAppliedPropertySetters(Control control)
```

Comment:
Does this is limited to `Setters` ? For instance, if we use an `ObjectAnimationUsingKeyFrames` to apply a `ThemeRessource`, should we also re-evaluate it ?

Reply:
This code is completely removed now.